tools/sadguess

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+WORKDIR=`pwd`
+BASISDIR='../basis'
```

Comment:
Absolute paths are dangerous. We set QUICK_INSTALL and QUICK_BASIS in quick.rc. Better check if these are defined and use those.